plugins/field-angle/test/field_angle_test.mocha.js

```diff
@@ -0,0 +1,381 @@
+/**
+ * @license
+ * Copyright 2023 Google LLC
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+const {testHelpers} = require('@blockly/dev-tools');
+const {FieldAngle} = require('../src/index');
+const {assert} = require('chai');
+
+const {
```

Comment:
FYI I filed https://github.com/google/blockly-samples/issues/1676 about the type issues with the test helpers.